Docs/sphinx_documentation/source/LinearSolvers.rst

```diff
@@ -404,7 +404,8 @@ as living at face centroids, modify the setBCoeffs command to be
 External Solvers
 ================
 
-AMReX can use the `hypre <https://computing.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods>`_ algebraic multigrid solver, BoomerAMG, 
+AMReX can use the 
+`hypre <https://computing.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods>`_ algebraic multigrid solver, BoomerAMG, 
```

Comment:
@asalmgren recommend updating this to reflect that we can now use BoomerAMG, GMRES (all variants), PCG, BiCGSTAB,  from hypre as main solver. And BoomerAMG and Euclid as preconditioners.

Reply:
I've added those -- thanks for pointing that out!